api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -2744,7 +2745,8 @@ public void saveOrder_shouldFailIfTheJavaTypeOfThePreviousOrderDoesNotMatch() th
 		Metadata metaData = new MetadataSources(standardRegistry).addAnnotatedClass(Allergy.class)
 				.addAnnotatedClass(Encounter.class).addAnnotatedClass(SomeTestOrder.class)
 				.addAnnotatedClass(Diagnosis.class).addAnnotatedClass(Condition.class)
-				.addAnnotatedClass(Visit.class).getMetadataBuilder().build();
```

Comment:
@alinmihaila99 what could be the reason behind this change?

Reply:
@gitcliff  that change was necessary to make sure that EncounterProvider is also added to the list of the annotated classes,checkout the code snipet below that